includes/class-sensei-admin.php

```diff
@@ -197,24 +194,39 @@ public function admin_menu_highlight() {
 		}
 
 		// phpcs:disable WordPress.WP.GlobalVariablesOverride.Prohibited -- Only way to highlight our special pages in menu.
-		if ( $screen->base == 'post' && $post_type == 'course' ) {
-
-			$parent_file = 'edit.php?post_type=course';
-
-		} elseif ( $screen->base == 'edit-tags' && $taxonomy == 'course-category' ) {
-
-			$submenu_file = 'edit-tags.php?taxonomy=course-category&amp;post_type=course';
-			$parent_file  = 'edit.php?post_type=course';
-
-		} elseif ( $screen->base == 'edit-tags' && $taxonomy == 'module' ) {
-
+		if ( 'edit-tags' === $screen->base && 'module' === $taxonomy ) {
+			$parent_file  = 'sensei';
 			$submenu_file = 'edit-tags.php?taxonomy=module&post_type=course';
-			$parent_file  = 'edit.php?post_type=course';
 
-		} elseif ( in_array( $screen->id, array( 'sensei_message', 'edit-sensei_message' ) ) ) {
-
-			$submenu_file = 'edit.php?post_type=sensei_message';
-			$parent_file  = 'sensei';
+		} else if ( in_array( $screen->id, [ 'edit-module', 'admin_page_module-order' ], true ) ) {
+			// Module pages.
+			$parent_file              = 'sensei';
+			$_wp_real_parent_file[''] = 'sensei';
```

Comment:
We needed to set the `$_wp_real_parent_file` because of this issue: https://core.trac.wordpress.org/ticket/40649